### ISSUE TITLE
Customizing Appearance

### ISSUE THREAD

**Issue by 0xJeu** [UdaraJay/Pile#45]
Hi, I Love the app. Can you get themes going on the app? something that would enable unique planes for every user. Eventually, a theme gallery would slowly build up.

I am open to any ideas to help build out this idea or whatever comes out of this.

**Comment by 0xdhrv** [comment]
Yes, I agree. Color themes might come little later, but fonts could be quite useful. My notes are based ligatures.
